specs/integrations/synthetics.md

```diff
@@ -0,0 +1,42 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent takes the responsibility of creating the [`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent) header for each outgoing network request associated with a test during every monitor execution.
```

Comment:
What I'm missing here is a brief description on how/when synthetics is generating the trace id. For example, does it create a unique id for each step in a monitor execution or is there a shared id for a single monitor execution? Is the trace id stored as a metadata in for the monitor execution results or only in the network requests?

Reply:
Added some extra details, Let me know if missed something https://github.com/elastic/apm/pull/825/commits/dadb42b77b23b117dfb575a36723f50d711c6d5e